### ISSUE TITLE
Question: Using Cell Level Binding with sap.ui.table.Table

### ISSUE THREAD

**Issue by LadwigDev** [abap2UI5/abap2UI5#787]
Hi, I have a question how to pass the column name and the row index into the event arguments when using sap.ui.table.Table:

In Demo 144 you are defining input fields outside the sap.m.table which are linked to the cells in the table. In my app I would also like to pass column and row to the event, but there is no loop through the rows and no additional input fields. I am only able to pass the column name to the event argument table but not the current row.

`DATA(table) = page->flex_box( height = '85vh' )->ui_table( alternaterowcolors = 'true' visiblerowcountmode = 'Auto' fixedrowcount = '1' selectionmode = 'None'  rows = client->_bind_edit( mt_output ) ).`

`DATA(columns) = table->ui_columns( ).`

`columns->ui_column( width = '5rem' sortproperty = 'PL_TOTAL' filterproperty = 'PL_TOTAL' )->text( text = 'Plan' )->ui_template( )->input( value = `{PL_TOTAL}` submit = client->_event( val = 'PL_TOTAL_CHANGE' t_arg = lt_event_arguments ) editable = abap_true type = 'Number' ).`

Is there a possibility to access the table row and pass it to the argument table so that I am able to know in which of the rows the user fired the input submit event?

Best regards
Jan

**Comment by oblomov-dev** [comment]
Hi @LadwigDev,

it seems you use the importing parameter t_arg not correctly.

Have you checked demo 48, there it is used with:
```
  ( n = 'detailPress'      v = client->_event( val = 'EDIT' t_arg = VALUE #( ( `${TITLE}`  )  ) ) )
```
I think you can adjust this line to make it work for your use case.

Or check demo 52, there you can see how to access parameters of the event source:
```
  lo_cells->link( text = '{PRODUCT}' press = client->_event( val = `POPOVER_DETAIL` t_arg = VALUE #( ( `${$source>/id}` ) ( `${PRODUCT}` ) )  ) ).
```

Does this help you? Otherwise feel free to create a little demo app with your problem which can be copy&pasted, that makes it easier to find a solution.

Best regards.

**Comment by LadwigDev** [comment]
Hi @oblomov-dev,
thanks for the tip! I did not know yet that you can address the html attributes. However, I would need to pass the id attribute of the fourth parent element to the event in order to find the correct column and row.

If I use ${source>/id} as argument I am receiving the value of the input element and I would need the one of the td element (s. screenshot). Is there a way to do it?

<img width="1083" alt="Element" src="https://github.com/abap2UI5/abap2UI5/assets/155659152/5bda15e1-104d-4b30-bee9-949563574197">

**Comment by oblomov-dev** [comment]
HI @LadwigDev,

mhh... never had this use case before, maybe it is possible to assign this values somehow, but i need to check this. 

Can you create a little example app which reproduces your use case and paste the source code here? Then i can copy & paste it to my system and try to find a solution.

Best regards.

**Comment by LadwigDev** [comment]
Hi  @oblomov-dev,
if you have the time for doing that I would really appreciate it :)

I have attached a test app to this post. I have added the input event to column 5. The goal would be: If you type in a number there and press "Enter" you are able to automatically calculate other cells in the same row based on the input value.

Best regards.

[Test App.txt](https://github.com/abap2UI5/abap2UI5/files/13964404/Test.App.txt)
